layout/settings/settings_interface.xml

```diff
@@ -675,7 +689,7 @@
 
 				<Panel class="settings-group__header">
 
-					<Label class="settings-group__title" text="#Settings_Synchronizer_Title" />
+					<Label class="settings-group__title" text="#Settings_Synchronizer_Title" tags="sync,strafe" />
```

Comment:
I think this should also have a "trainer" tag.

Reply:
Good call, though I'm on my laptop and about to go to bed. If you wanna add this to this branch this resolve this, be my guest, would be good to get in today

Reply:
Added that TAG